recursive_count_th/count_th.py

```diff
@@ -6,10 +6,10 @@
 
 
 def count_th(word):
-    substring = 'th'
-    count = word.count(substring)
-
-    if count == 0:
+    substring = "th"
+    if len(word) != = substring:
         return
+    for i in range(0, len(word)):
+        if i == substring:
 
-    return count
+    return substring += 1
```

Comment:
There are several syntax errors here, so the code will not run.

Even if you can't get working code, try to type out your plan of the problem. If you can find a full plan in pseudocode, that will help you once you start coding in the actual language, no matter if it's python, JS, etc